installation/production_deb.rst

```diff
@@ -104,7 +104,7 @@ shell should output the worker nodes we added to the pg_dist_node table above.
 
 **Ready to use Citus**
 
-At this step, you have completed the installation process and are ready to use your Citus cluster. To help you get started, we have a :ref:`tutorial<tut_hash>` which has instructions on setting up a Citus cluster with sample data in minutes.
+At this step, you have completed the installation process and are ready to use your Citus cluster. To help you get started, we have a :ref:`tutorial<multi_tenant_tutorial>` which has instructions on setting up a Citus cluster with sample data in minutes.
```

Comment:
@samay-sharma / @begriffs -- I had two questions.

1. If we link to the tutorial from Debian / RHEL pages, what do you think about adding steps for "localhost" in a future iteration?
2. Why do we point to our tutorial from production install pages but not from others (including Docker)?